lib/seattleflu/db/cli/command/redcap_det.py

```diff
@@ -33,8 +33,8 @@ def redcap_det():
     required = True)
 @click.option("--token-name",
     metavar = "<token-name>",
-    help = "The name of the environment variable that holds the API token for the <project-id>",
-    required = True)
+    help = "The name of the environment variable that holds the API token",
+    default = "REDCAP_API_TOKEN")
```

Comment:
Oh, I realized: I think all projects share the same API Token. It's peculiarly only accessible through the Kiosks project, but it is the same one used for Shelters.

So we may not need this option at all.

Reply:
Each project needs a different API Token, the API Token is actually how REDCap knows which project to access records from.

Reply:
Oh... do you know how we go about requesting API tokens for the other projects?

Reply:
If you have API access, there will be "API" and "API Playground" links in the sidebar nav for the project. The first page will let you generate a token for the project.